client/src/components/Button/Button.stories.js

```diff
@@ -1,7 +1,15 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-
+import './Button.css'
 import Button from './Button';
 
+function myFunction (){
+    alert("details about dress");
+}  
+
 storiesOf('Button', module)
     .add('Primary', () => <Button>This is a Button</Button>)
+    .add('Details', () => <Button className="info"onclick={myFunction()}>+</Button>)
```

Comment:
You should not invoke the function inside the `stories.js` file. You should just pass the function down as props and then use it inside the `Button` component